modules/table/src/main/java/org/apache/ignite/internal/table/KeyValueViewImpl.java

```diff
@@ -236,7 +242,11 @@ public void putAll(@Nullable Transaction tx, Map<K, V> pairs) {
     public CompletableFuture<Void> putAllAsync(@Nullable Transaction tx, Map<K, V> pairs) {
         Objects.requireNonNull(pairs);
         for (Entry<K, V> entry : pairs.entrySet()) {
-            Objects.requireNonNull(entry.getKey());
+            K key = entry.getKey();
+            V val = entry.getValue();
+
+            Objects.requireNonNull(key);
```

Comment:
https://github.com/apache/ignite-3/pull/3714/files#r1596527346

Reply:
Added variable names to `requireNonNull` calls.